### ISSUE TITLE
glob patterns are not supported when using as a module

### ISSUE THREAD

**Issue by joshvito** [addyosmani/a11y#80]
```
// file: test-a11y.js
const a11y = require('a11y');
const _glob = './src/**/*.html';
const options = {
    viewportSize: '800x600'
};

a11y(_glob, options, (err, reports) => {
    if (err) console.log(err);
    const audit = reports.audit; // `a11y` Formatted report
    const report = reports.report; // DevTools Accessibility Audit formatted report
});

```
The above file, when ran (`node test-a11y.js`), will always fail with the following error. `Couldn't load url: "./src/**/*.html"`
